### DISCUSSION TITLE
GitHub Page

### DISCUSSION THREAD

**Discussion by JPJimrah** [community/community#27513] [New to GitHub] [answered]
`My` work appears fine locally, but after publishing the page on GitHub, but the last button in the index.html file appears different from others. What's the way forward? I need urgent help, please.
This is the page link: https://jpjimrah.github.io/CSS-grid-and-flexbox/
This is the repository link: https://github.com/JPJimrah/CSS-grid-and-flexbox

**Comment by sawyerbivens** [comment]
What does your work look like locally?

**Comment by jfullstackdev** [comment, marked as answer]
no errors at the Developer tools so I expect all files loaded correctly 

![image](https://user-images.githubusercontent.com/47092464/181672424-486c9939-f4bf-4078-b22c-a81f7a88b0da.png)

what makes it different from your local deployment ?

**Reply by sawyerbivens** [reply]
For me both locally and on GH pages things look the exact same

![image](https://user-images.githubusercontent.com/48366954/181691268-b532b711-d472-46af-82c0-32bb40af3990.png)

Could I see the differences in what you see so I can try to better understand the issue?

**Comment by JPJimrah** [comment]
I made some adjustments. It appears fine on desktop.

**Comment by blackbirdcoder** [comment]
You need to look at the markup

**Comment by blackbirdcoder** [comment]
Pay attention to markup responsiveness
